extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/ConflictsWithScrutinizer.java

```diff
@@ -3,18 +3,50 @@
 import org.openrefine.wikidata.qa.QAWarning;
 import org.openrefine.wikidata.updates.ItemUpdate;
 import org.wikidata.wdtk.datamodel.interfaces.PropertyIdValue;
+import org.wikidata.wdtk.datamodel.interfaces.Snak;
+import org.wikidata.wdtk.datamodel.interfaces.SnakGroup;
 import org.wikidata.wdtk.datamodel.interfaces.Statement;
 import org.wikidata.wdtk.datamodel.interfaces.Value;
 
-import java.util.Set;
-import java.util.Map;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
 
 public class ConflictsWithScrutinizer extends EditScrutinizer {
 
     public static final String type = "having-conflicts-with-statements";
+    public static String CONFLICTS_WITH_CONSTRAINT_QID = "Q21502838";
+    public static String CONFLICTS_WITH_PROPERTY_PID = "P2306";
+    public static String ITEM_OF_PROPERTY_CONSTRAINT_PID = "P2305";
+
+    class ConflictsWithConstraint {
+        PropertyIdValue conflictingPid;
+        List<Value> itemList;
+
+        ConflictsWithConstraint(Statement statement) {
+            this.setParams(statement);
```

Comment:
I would inline the `setParams` method (pull its contents into the constructor and delete the method), because we do not expect to have to change the constraint definition after constructing it. In general, when we can make immutable classes, it is a good idea to build them as such (so you could also mark the two fields above as final).